src/util/storagePoolForm.tsx

```diff
@@ -4,26 +4,25 @@ import type {
 } from "types/storage";
 import { StoragePoolFormValues } from "pages/storage/forms/StoragePoolForm";
 import { ClusterSpecificValues } from "components/ClusterSpecificSelect";
+import { zfsDriver, btrfsDriver, lvmDriver } from "util/storageOptions";
 
 export const toStoragePoolFormValues = (
   pool: LxdStoragePool,
   poolOnMembers?: LXDStoragePoolOnClusterMember[],
 ): StoragePoolFormValues => {
   const sourcePerClusterMember: ClusterSpecificValues = {};
-  poolOnMembers?.forEach(
-    (item) =>
-      (sourcePerClusterMember[item.memberName] = item.config?.source ?? ""),
-  );
+  const sizePerClusterMember: ClusterSpecificValues = {};
+  const hasSize = zfsDriver || lvmDriver || btrfsDriver;
+
+  poolOnMembers?.forEach((item) => {
+    if (item.driver === hasSize) {
+      sizePerClusterMember[item.memberName] = item.config?.size ?? "";
+    }
```

Comment:
```suggestion
  const isStoragePoolWithSize = (driver: string) => {
    const driversWithSize = [zfsDriver, lvmDriver, btrfsDriver];
    return driversWithSize.includes(driver);
  }

  poolOnMembers?.forEach((item) => {
    if (isStoragePoolWithSize(item.driver)) {
      sizePerClusterMember[item.memberName] = item.config?.size ?? "";
    }
```
I think this might be what was meant here? 

Probably best to move the `isStoragePoolWithSize` function into a util file, to reuse it in `ClusteredDiskSizeSelector`. That avoids introducing two places that define which drivers support the size configuration.

Reply:
storagePoolForm is actually already a util file. Should it stay within?